src/io/perun.clj

```diff
@@ -284,11 +295,18 @@
         (reset! prev fileset)
         pod))))
 
-(defn- render-in-pod [pod sym global-meta file]
+(defn- render-in-pod [pod sym global-meta file-meta]
   {:pre [(symbol? sym) (namespace sym)]}
-  (pod/with-eval-in pod
-    (require '~(symbol (namespace sym)))
-    ((resolve '~sym) ~global-meta ~file)))
+  ;; Ensure passed seqs are vectors, otherwise lists/array-seqs etc
+  ;; wrapped in parentheses will be interpreted as function calls
+  (let [m (if (sequential? file-meta) (vec file-meta) file-meta)]
+    (pod/with-eval-in pod
+      (require '~(symbol (namespace sym)))
+      ((resolve '~sym) ~global-meta ~m))))
```

Comment:
This got me a couple of times when passing filtered lists to the `render-in-pod` function.